apps/tmbitwarden.sh

```diff
@@ -84,7 +84,10 @@ check_command ./bitwarden.sh rebuild
 check_command ./bitwarden.sh start
 if check_command ./bitwarden.sh updatedb
 then
-msg_box "Bitwarden was sucessfully installed! Please visit $(grep 'url:' /root/bwdata/config.yml | awk '{print$2}'):8443 to setup your account."
+msg_box "Bitwarden was sucessfully installed! Please visit $(grep 'url:' /root/bwdata/config.yml | awk '{print$2}'):8443 to setup your account.
+
+After the account it setup, please disable user registration by running sudo bash $SCRIPTS/menu.sh and choose:
+Server Configuration --> Bitwarden Registration"
```

Comment:
I am unsure if it fits that well in server_configuration, since it is configuring the bitwarden app.
Are you sure with that?

Reply:
Also this would be an option before installing any apps if executed during the startup script, isn't that a problem?

Reply:
> I am unsure if it fits that well in server_configuration

OK, so where would you put it?

> Also this would be an option before installing any apps if executed during the startup script, isn't that a problem?

I don't get this question, sorry. I see it as a more polite thing that like "hey, you maybe would like to disable Public User Registration once we're done here"

Reply:
> OK, so where would you put it?

Hm, maybe as an additional option for an apps_script?
or maybe the bitwarden option in additional_apps itself could be a menu, where you can choose between the installer/reinstaller/uninstall script and the bitwarden configuration script?

> I don't get this question, sorry. I see it as a more polite thing that like "hey, you maybe would like to disable Public User Registration once we're done here"

What I mean is, that if you run the startup script, you first see the server_configuration menu with the option to configure bitwarden. The thing is, that you will see the additional_apps menu afterwards where you have the first chance to install bitwarden.

Reply:
True!

Let's change it to the apps menu instead. Do you have time? I'm in the middle of a big migration.

Please also change the script itself to a radiolist as you proposed.

Thanks! :+1:

Reply:
I've changed it to the apps menu and changed the checklist to a radiolist